### ISSUE TITLE
Transaction Support For MongoDB 4.0?

### ISSUE THREAD

**Issue by JRose6** [nesdis/djongo#228]
Are there plans to add support for MongoDB 4.0 Transactions using Django's transaction capabilities?

**Comment by nesdis** [comment]
Yes this is in the pipeline. But delivery date depends on several factors

**Comment by williamborba** [comment]
awaiting this feature

https://www.mongodb.com/blog/post/multi-document-transactions-in-mongodb

**Comment by florealcab** [comment]
I am waiting for it too

**Comment by talwrii** [comment]
This appears to be implemented: https://nesdis.github.io/djongo/djongonxt-database-transactions/

**Comment by florealcab** [comment]
> This appears to be implemented: https://nesdis.github.io/djongo/djongonxt-database-transactions/

Only in djongoNxt :(

**Comment by gwanghyeongim** [comment]
Still not supported?
